spec/lib/popular_article_spec.rb

```diff
@@ -0,0 +1,60 @@
+require File.join(Rails.root, 'lib', 'popular_article')
+
+describe PopularArticle do
+  let!(:blog) { create(:blog) }
+
+  subject { described_class }
+
+  describe '#find' do
```

Comment:
In an ideal world I'd do so though the code base needs some love in generally.

Maybe that's the best time to look at such things :).